gcp/modules/gpii-preferences/resources/preferences-couchdb-network-policy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: preferences-couchdb-network-policy
+  namespace: gpii
+spec:
+  podSelector:
+    matchLabels:
+      app: couchdb
```

Comment:
As above, should this be under `couchdb/resources`?

Reply:
This discussion is continued on the other thread.